addons/mail/models/mail_thread.py

```diff
@@ -2595,11 +2595,6 @@ def _notify_classify_recipients(self, recipient_data, model_name, msg_vals=None)
         groups = self._notify_get_groups(msg_vals=local_msg_vals)
         access_link = self._notify_get_action_link('view', **local_msg_vals)
 
-        if model_name:
-            view_title = _('View %s', model_name)
```

Comment:
if view_title is removed, there should be an error (because it's undefined) in the line below:

```diff
group_button_access.setdefault('title', view_title)
```